test/sequential/rolldown.withdraw.fees.test.ts

```diff
@@ -53,73 +55,120 @@ beforeEach(async () => {
 });
 
 test("GIVEN a withdrawal, WHEN paying with GASP and withdrawing GASP, some fees goes to treasury", async () => {
-  const treasuryBalanceBefore = await api.query.tokens.accounts(
-    treasuryAccount,
-    GASP_ASSET_ID,
+  const treasuryBalanceBefore = stringToBN(
+    (await getTokensAccountInfo(treasuryAccount, GASP_ASSET_ID)).free,
   );
   await Sudo.batchAsSudoFinalized(Assets.mintNative(testUser));
+  const userBalanceBefore = stringToBN(
+    (await getTokensAccountInfo(testUser.keyRingPair.address, GASP_ASSET_ID))
+      .free,
+  );
+
   const events = await signTx(
     getApi(),
     await Withdraw(testUser, withdrawalAmount, gaspIdL1Asset.ethereum, chain),
     testUser.keyRingPair,
   );
-  const eventFiltered = filterZeroEventData(events, "Transfer");
-  const transferAmountBefore = eventFiltered.amount.replaceAll(",", "");
-  const transferAmount = new BN(transferAmountBefore);
-  expect(eventFiltered.from).toEqual(testUser.keyRingPair.address);
-  expect(eventFiltered.to).toEqual(treasuryAccount);
-  const treasuryBalanceAfter = await api.query.tokens.accounts(
-    treasuryAccount,
-    GASP_ASSET_ID,
+  const eventResponse = getEventResultFromMangataTx(events);
+  expect(eventResponse.state).toEqual(ExtrinsicResult.ExtrinsicSuccess);
+
+  const transferEvent = filterZeroEventData(events, "Transfer");
+  const withdrawalFee = stringToBN(transferEvent.amount);
+  expect(transferEvent.from).toEqual(testUser.keyRingPair.address);
+  expect(transferEvent.to).toEqual(treasuryAccount);
+  const depositEvent = filterZeroEventData(events, "Deposited");
+  const networkFees = stringToBN(depositEvent.amount);
```

Comment:
This can be missleading, networkFees are calculated by .PaymentInfo method in a Tx, ie: 
```
const tx = api.tx.proxy.proxy(
      user1.keyRingPair.address,
      undefined,
      api.tx.xyk.compoundRewards(8, BN_MILLION),
    );
    const encodedCall = tx.unwrap().toHex();
    const weight = await tx
      .paymentInfo(user1.keyRingPair)
      .then((value) => value.weight);

```

Reply:
Changed to transactionFee